api/v1alpha2/awscluster_conversion.go

```diff
@@ -44,8 +46,15 @@ func (src *AWSCluster) ConvertTo(dstRaw conversion.Hub) error { // nolint
 	if ok, err := utilconversion.UnmarshalData(src, restored); err != nil || !ok {
 		return err
 	}
+
 	dst.Spec.ImageLookupOrg = restored.Spec.ImageLookupOrg
-	dst.Spec.ControlPlaneLoadBalancer.CrossZoneLoadBalancing = restored.Spec.ControlPlaneLoadBalancer.CrossZoneLoadBalancing
+	dst.Spec.ImageLookupBaseOS = restored.Spec.ImageLookupBaseOS
+	if restored.Spec.ControlPlaneLoadBalancer != nil {
+		restored.Spec.ControlPlaneLoadBalancer.DeepCopyInto(dst.Spec.ControlPlaneLoadBalancer)
```

Comment:
Why not `=` assignment - the restored data came from unmarhshaling the json in the annotation, not from the cache?

Reply:
Thanks, I changed it last night and forgot to change it back